### DISCUSSION TITLE
"player.c:2708" Delay error -524 when checking running latency.

### DISCUSSION THREAD

**Discussion by sberl** [mikebrady/shairport-sync#1930] [Q&A]
I just installed shairport-sync on a fresh install of Raspberry OS on a RPi 4.
Version String: "4.3.4-1-g910264e5-AirPlay2-smi10-OpenSSL-Avahi-ALSA-soxr-sysconfdir:/etc"

It works fine when started on the command line. Can play music from phone or laptop and it comes out the headphone jack of the Pi just fine.

When running as a service, the log shows the normal startup messages (same as what I see on command line start), but as soon as my phone attaches, it starts to spew out lots of these messages. 

> "player.c:2708" Delay error -524 when checking running latency.

No sound comes from the Pi headphone jack in this case.

Any ideas what's wrong and how to fix it?

**Comment by mikebrady** [comment]
Thanks for the post. This should really be an [Issue](https://github.com/mikebrady/shairport-sync/issues/new/choose), TBH.

Anyway, two things jump out before we have any more info. Error 524 is mysterious, but it seems to be related to HDMI connections being idle. Does that make any sense in your situation?

The second is the general point that the environment that the service runs in is quite different to the logged-in user, especially if you are using a Pi with a GUI.

SO, if you could make this an Issue (so to speak!) it would help.

**Comment by mikebrady** [comment]
Moved to this [Issue](https://github.com/mikebrady/shairport-sync/issues/1931).
